test/controllers/survey_controller_test.exs

```diff
@@ -676,13 +676,13 @@ defmodule Ask.SurveyControllerTest do
       assert json_response(conn, 422)["errors"] != %{cutoff: "must be less than #{max_int}"}
     end
 
-    test "rejects update with correct error when cutoff field is less than zero", %{conn: conn, user: user} do
+    test "rejects update with correct error when cutoff field is less than -1", %{conn: conn, user: user} do
       project = create_project_for_user(user)
       survey = insert(:survey, project: project)
 
-      conn = put conn, project_survey_path(conn, :update, survey.project, survey), survey: Map.merge(@valid_attrs, %{cutoff: 0})
+      conn = put conn, project_survey_path(conn, :update, survey.project, survey), survey: Map.merge(@valid_attrs, %{cutoff: -1})
 
-      assert json_response(conn, 422)["errors"] != %{cutoff: "must be greater than 0"}
+      assert json_response(conn, 422)["errors"] != %{cutoff: "must be greater than -1"}
```

Comment:
The validation message will make much more sense if you apply the change suggested in `survey.ex`